docs/src/develop/components/StatusDoc.js

```diff
@@ -40,10 +42,10 @@ var StatusDoc = React.createClass({
 
           <h3>Error</h3>
           <div className="example">
-            <Status value="error"/>
+            <Status value="error" a11yTitle="critical"/>
```

Comment:
I like you are concerned about updating the docs whenever you are doing changes :+1: